go/vt/vtgate/evalengine/translate_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Vitess Authors.
```

Comment:
Should we change the copyright of existing file?

Reply:
I think so? It's always safe to bump our own copyrights as long as we're actively working on the specific files (which we are in this PR!).

Reply:
actually we should not change the copyright year. if there are substantial changes you _could_ add the new copyright year, but that is not necessary.